PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/META-INF/MANIFEST.MF

```diff
@@ -14,7 +14,13 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.jst.j2ee;bundle-version="1.1.802";resolution:=optional,
  org.eclipse.jst.j2ee.web;bundle-version="1.1.810";resolution:=optional,
  com.microsoft.azuretools.sdk;bundle-version="3.0.6",
- com.jcraft.jsch;visibility:=reexport
+ com.jcraft.jsch;visibility:=reexport,
+ org.eclipse.equinox.p2.ui,
```

Comment:
Please check the UI I pasted in GitHub

---
In reply to: [132111214](https://github.com/Microsoft/azure-tools-for-java/pull/845#discussion_r132111214) [](ancestors = 132111214)